spec/test_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "vcr"
+require "webmock"
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/cassettes"
+  config.hook_into :webmock
+  config.configure_rspec_metadata!
+  config.filter_sensitive_data("<GITHUB_TOKEN>") { gh_test_access_token }
+  config.filter_sensitive_data("<GITHUB_REPOSITORY>") { gh_test_repository }
+end
+
+def gh_test_access_token
+  ENV.fetch("GITHUB_TEST_TOKEN", "x" * 40)
+end
+
+def gh_test_repository
+  ENV.fetch("GITHUB_TEST_REPOSITORY", "krauselukas/test_github_deployments")
```

Comment:
here should be the official project, right?

Reply:
I would not spam the official project with test data, if we point the test to 'openSUSE/open-build-service' all the deployments created during the test will appear there. This also might break the locks that are in place. Maybe better we create one central test repo for the future

Reply:
ah or do you mean the "openSUSE/obs_github_deployments repo" ?